patches/boringssl/enable_x509_v_flag_trusted_first_flag.patch

```diff
@@ -0,0 +1,20 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Juan Cruz Viotti <jv@jviotti.com>
+Date: Thu, 30 Sep 2021 13:39:23 -0400
+Subject: Enable X509_V_FLAG_TRUSTED_FIRST flag
+
+Signed-off-by: Juan Cruz Viotti <jv@jviotti.com>
+
+diff --git a/crypto/x509/x509_vpm.c b/crypto/x509/x509_vpm.c
+index 5a881d64c30076404cc800fff9e943bb0b30d2ac..29d5341efc8eb7ae6f90bdde5a8032e99f75c98e 100644
+--- a/crypto/x509/x509_vpm.c
++++ b/crypto/x509/x509_vpm.c
+@@ -528,7 +528,7 @@ static const X509_VERIFY_PARAM default_table[] = {
+      (char *)"default",         /* X509 default parameters */
+      0,                         /* Check time */
+      0,                         /* internal flags */
+-     0,                         /* flags */
++     X509_V_FLAG_TRUSTED_FIRST, /* flags */
```

Comment:
This value [is still 0 upstream](https://source.chromium.org/chromium/chromium/src/+/main:third_party/boringssl/src/crypto/x509/x509_vpm.c;l=531?q=X509_VERIFY_PARAM&ss=chromium), so I'm wondering if upstream has a different fix or is otherwise unaffected...?

Reply:
@ckerr We haven't checked, but this is the *default* values to BoringSSL. Other parts of Chromium might be overwriting this default?

Reply:
@ckerr This flag is actually set by default in Node.js: https://github.com/nodejs/node/blob/master/deps/openssl/openssl/crypto/x509/x509_vpm.c#L479

Reply:
@jviotti nicely done. Thanks for the clarification!